runtime/oti/j9nonbuilder.h

```diff
@@ -5151,6 +5210,11 @@ typedef struct J9VMThread {
 	struct PortlibPTBuffers_struct* ptBuffers;
 	j9object_t blockingEnterObject;
 	void* gcExtensions;
+	UDATA contiguousIndexableHeaderSize;
+	UDATA discontiguousIndexableHeaderSize;
+#if defined(J9VM_ENV_DATA64)
+	UDATA isIndexableDataAddrPresent;
```

Comment:
Perhaps
```
BOOLEAN isIndexableDataAddrPresent;
```

Reply:
Do we need to add this here since it can be accessed through `J9JavaVM` as well?

Reply:
Do we need to add this here since it can be accessed through `J9JavaVM`?

Reply:
This is still `UDATA`: why?

Reply:
This was for DDR compatibility reasons. A few checks were not working as expected when `BOOLEAN` was used, perhaps because DDR didn't recognize it as a type, or for some other reason.

Reply:
@VermaSh Yes this was added here intentionally and is often accessed in several places through `J9VMThread`, similar to many other fields that are contained in both `J9JavaVM` and `J9VMThread`.